hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/volume/CreateVolumeHandler.java

```diff
@@ -40,10 +41,13 @@
       description = "Owner of the volume")
   private String ownerName;
 
+  @Option(names = {"--spaceQuota", "-s"},
```

Comment:
-s -> -sq

Reply:
The abbreviation of this parameter was previously [confirmed with Yisheng](https://github.com/apache/hadoop-ozone/pull/1233#discussion_r458723395). In general, the parameter abbreviation is a single letter, in order to keep the same with other parameters here sq->s.
Of course, For better understanding, I added a detailed description.

Reply:
I'm fine with the comment with @ChenSammi.
Let's update it.

Reply:
Agreed, I will fix the naming problem.